drivers/led/issi/is31fl3218.c

```diff
@@ -45,9 +45,7 @@ void IS31FL3218_write_register(uint8_t reg, uint8_t data) {
 
 void IS31FL3218_write_pwm_buffer(uint8_t *pwm_buffer) {
     g_twi_transfer_buffer[0] = ISSI_REG_PWM;
-    for (int i = 0; i < 18; i++) {
-        g_twi_transfer_buffer[1 + i] = pwm_buffer[i];
-    }
+    memcpy(g_twi_transfer_buffer + 1, pwm_buffer + i, 18);
```

Comment:
Original code didn't have an offset, and `i` isn't defined here, perhaps a typo from copy-pasting code from other files?
```suggestion
    memcpy(g_twi_transfer_buffer + 1, pwm_buffer, 18);
```

Reply:
Ah yep thanks! I remember looking at it. I must have fixed it on another branch...